cartoonfight/player.py

```diff
@@ -5,6 +5,7 @@
 class Player(object):
 
     def __init__(self, position, maxHealth, size, isPlayerOne, display):
+        self.name = ' '
```

Comment:
I think its better to leave this variable initialized with empty strings rather than a blank space
`self.name = ''`